cmd/kubectl-gadget/bcck8s.go

```diff
@@ -53,7 +53,7 @@ var tcptopCmd = &cobra.Command{
 
 var tcpconnectCmd = &cobra.Command{
 	Use:   "tcpconnect",
-	Short: "Suggest Kubernetes Network Policies",
+	Short: "Trace TCP active connections",
```

Comment:
I think "Trace TCP active connections" could be a bit misleading as "active" refers to connections created with the "connect()" syscall (instead of accept()" and not to connections that are active at the moment. 

What about "Trace TCP connect() system calls" ?

Reply:
Sure, done.